tests/utils.py

```diff
@@ -21,6 +21,18 @@
     'source': 'test'
 }
 
+POSTGRES_RAW_DOC = {
```

Comment:
This was done just cause the doc item had to be different (bytes vs not) but I can definitely make this less verbose